Cargo.toml

```diff
@@ -33,6 +33,27 @@ default-members = [
 # features without this, causing forced "std" in downstream crate.
 resolver = "2"
 
+[workspace.package]
+version = "0.4.0-alpha.1"
+repository = "https://github.com/openwsn-berkeley/edhoc-rs/"
+license = "BSD"
```

Comment:
Note that this is plainly moving data around from the other crates. If you want to have #152 resolved in here right away, I'm happy to update this, but would prefer explicit confirmation in that #152 that it is to be resolved toward which-ever license (in addition to one of you merging this, which'll need to happen anyway).

Reply:
Confirming BSD-3-Clause is the right license.